db/migrate/20210420015635_add_searchable_to_user_fields.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddSearchableToUserFields < ActiveRecord::Migration[6.0]
+  def change
+    add_column :user_fields, :searchable, :boolean, default: :false
```

Comment:
I recommend every boolean should be `null: false` since it just causes problems and is basically the same as `false`.